### DISCUSSION TITLE
[FEAT] Create a Favicon

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [collab-community/journey-book#75] [Ideas]
- For the website, we need a favicon, hence I require someone to create a favicon or to share a favicon design from the internet (copyright free) with me.

Was thinking of the favicon being a book or something.

Any ideas?

cc: @vikasganiga05 @Adarsh-jaiss

**Comment by mehakagrawal** [comment]
> * For the website, we need a favicon, hence I require someone to create a favicon or to share a favicon design from the internet (copyright free) with me.
> 
> Was thinking of the favicon being a book or something.
> 
> Any ideas?
> 
> cc: @vikasganiga05 @Adarsh-jaiss

There are many online favicon generators. You can generate one as per your needs. Here's one I found: https://www.favicon-generator.org/search/---/Book

I hope it helps!

**Comment by KendallDoesCoding** [comment]
> > * For the website, we need a favicon, hence I require someone to create a favicon or to share a favicon design from the internet (copyright free) with me.
> > 
> > Was thinking of the favicon being a book or something.
> > Any ideas?
> > cc: @vikasganiga05 @Adarsh-jaiss
> 
> There are many online favicon generators. You can generate one as per your needs. Here's one I found: https://www.favicon-generator.org/search/---/Book
> 
> I hope it helps!

Hey Mehak,

Yeah, I know this, but I would prefer if someone makes the favicon, because there are all weird copyright issues these days.

**Comment by Adarsh-jaiss** [comment]
-- @mehakagrawal can you please make one favicon for us

**Comment by mehakagrawal** [comment]
Hey, I have added favicon images in this drive folder - https://drive.google.com/drive/folders/1hnBWaP-MD5mnM61Yn87jwiShjxDoJH6b?usp=sharing

Let me know your thoughts?

**Comment by KendallDoesCoding** [comment]
Hey Mehak,

LGTM, Please let me know your thoughts Vikas and Adarsh.

**Comment by vikasganiga05** [comment]
Hi Mehak, The icon is pixelated. Please can you share a good resolution image? We need to understand what the icon is saying :slightly_smiling_face: and Thank You for your time creating this icon :pray:

**Comment by mehakagrawal** [comment]
> Hi Mehak, The icon is pixelated. Please can you share a good resolution image? We need to understand what the icon is saying 🙂 and Thank You for your time creating this icon 🙏

Hi Vikas, it was my pleasure creating this icon :)
I have already added the png files of the icon in the same drive folder. Let me know if you don't find it there?

**Comment by vikasganiga05** [comment]
> 

Yes, I found it. But the PNG is the same size as ICO. No worries please can you explain what the icon says? Like what is in between the book?

**Comment by KendallDoesCoding** [comment]
Hi Vikas, I don't think it says anything, there are just a bunch of
avatars, let me know if this is correct Mehak.

**Comment by vikasganiga05** [comment]
> 

Oh! Okay. Then we wait for @Adarsh-jaiss thoughts on this.

**Comment by Abhinavkar** [comment]
Hey @KendallDoesCoding @Adarsh-jaiss  @vikasganiga05 Allow me to create some of icons for this project i will share you some couple of design i made 
Check this out this guys !!!
https://drive.google.com/drive/folders/1sAgcVPgDQa9zXesbLMkMOK8QEMVIhPx8?usp=sharing

**Reply by KendallDoesCoding** [reply]
> Hey @KendallDoesCoding @Adarsh-jaiss @vikasganiga05 Allow me to create some of icons for this project i will share you some couple of design i made Check this out this guys !!! https://drive.google.com/drive/folders/1sAgcVPgDQa9zXesbLMkMOK8QEMVIhPx8?usp=sharing

@Abhinavkar - please give me a .ico version of all these images at the earliest, also I have disqualified the following favicon from the poll as it doesn't suit the favicon for this project:

![Journey](https://user-images.githubusercontent.com/84712013/181733182-a9412e8b-11d1-4d38-8755-f39adb954bc0.png)

**Comment by KendallDoesCoding** [comment]
Hi Abhinav,

Thanks for wanting to contribute. Although, Mehak as already created
favicons for us. Will let you know if we need help. But, feel free to
contribute to this, by adding your journey, making the website better, etc.

Thanks a ton,
Kendall

**Comment by mehakagrawal** [comment]
> > 
> 
> Yes, I found it. But the PNG is the same size as ICO. No worries please can you explain what the icon says? Like what is in between the book?

There is a plant that is blooming above the pile of books.

**Comment by KendallDoesCoding** [comment]
> > > 
> > 
> > 
> > Yes, I found it. But the PNG is the same size as ICO. No worries please can you explain what the icon says? Like what is in between the book?
> 
> There is a plant that is blooming above the pile of books.

Ok cool. it looks good to me, if you like it @vikasganiga05 and @Adarsh-jaiss , we can proceed.

**Comment by Adarsh-jaiss** [comment]
-- yes I really loved it 💙😃

**Comment by KendallDoesCoding** [comment]
> -- yes I really loved it 💙😃

epic! let's proceed after @vikasganiga05 's approval in that case.

**Comment by vikasganiga05** [comment]
I like it too thanks! @mehakagrawal for creating favicon :pray:

**Comment by Abhinavkar** [comment]
Please Close this issue if the Favicon is created @KendallDoesCoding

**Comment by KendallDoesCoding** [comment]
Hi Abhinav,
The favicon is made but will close once is implemented on the website.

**Comment by KendallDoesCoding** [comment]
@mehakagrawak @Abhinavkar - I am transferring this into a discussion and will create a poll discussion, where we list all the favicons created by both of you for voting, which ever is voted the most will be our products favicon. Thanks


cc: @vikasganiga05 @Adarsh-jaiss @Vrashabh-Sontakke
